components/instrumentation/default/index.test.mjs

```diff
@@ -12,50 +13,50 @@ const normalizeContent = ({ content, ...rest }, source) => ({
   ...rest,
 });
 
-{
-  const file = {
-    url: "protocol://host/base/invalid.js",
-    content: "INVALID JAVASCRIPT",
-    type: "script",
-  };
-  assertThrow(
-    () =>
-      instrument(
-        createInstrumentation(
-          extendConfiguration(
-            createConfiguration("protocol://host/home/"),
-            {
-              "inline-source": false,
-              hooks: { apply: "$", eval: false },
-              packages: [
-                {
-                  path: "invalid.js",
-                  enabled: true,
-                },
-              ],
-            },
-            "protocol://host/base/",
-          ),
-        ),
-        file,
-        createMirrorMapping(file),
-      ),
-    /^ExternalAppmapError: Failed to parse js file$/u,
-  );
-}
+// {
+//   const file = {
+//     url: "protocol://host/base/invalid.js",
+//     content: "INVALID JAVASCRIPT",
+//     type: "script",
+//   };
+//   assertThrow(
+//     () =>
+//       instrument(
+//         createInstrumentation(
+//           extendConfiguration(
+//             createConfiguration("protocol://host/home/"),
+//             {
+//               "inline-source": false,
+//               hooks: { apply: "$", eval: false },
+//               packages: [
+//                 {
+//                   path: "invalid.js",
+//                   enabled: true,
+//                 },
+//               ],
+//             },
+//             "protocol://host/base/",
+//           ),
+//         ),
+//         file,
+//         createMirrorMapping(file),
+//       ),
+//     /^ExternalAppmapError: Failed to parse js file$/u,
+//   );
+// }
```

Comment:
Could use a comment on why it is commented out.

Reply:
Parsing is now done in the source component and should be tested here. I removed the commented out test because it was confusing (this is probably an oversight). rebased in b62efb0935dd303164a844d6b89b590e365d6a10